Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/MultiPositionController.cs

```diff
@@ -0,0 +1,1001 @@
+                checkNeutral = false;
+                bool isFirst = true;
+                string previous = "";
+                foreach (Position pair in PositionsList)
+                {
+                    if (pair.Type == currentPosition)
+                    {
+                        if (isFirst)
+                            break;
+                        currentPosition = previous;
+                        Locomotive.SignalEvent(Common.Event.MPCChangePosition);
+                        break;
+                    }
+                    isFirst = false;
+                    previous = pair.Type;
+                }
+            }
+            if (movement == Movement.Aft)
+            {
+                noKeyPressed = false;
+                checkNeutral = false;
+                bool selectNext = false;
+                foreach (Position pair in PositionsList)
+                {
+                    if (selectNext)
+                    {
+                        currentPosition = pair.Type;
+                        Locomotive.SignalEvent(Common.Event.MPCChangePosition);
+                        break;
+                    }
+                    if (pair.Type == currentPosition) selectNext = true;
+                }
+            }
+            if (movement == Movement.Neutral)
+            {
+                noKeyPressed = true;
+                foreach (Position pair in PositionsList)
+                {
+                    if (pair.Type == currentPosition)
+                    {
+                        if (pair.Flag.ToLower() == "springloadedbackwards" || pair.Flag.ToLower() == "springloadedforwards")
+                        {
+                            checkNeutral = true;
+                            elapsedSecondsFromLastChange = 0;
+                        }
+                        if (pair.Flag.ToLower() == "springloadedbackwardsimmediately" || pair.Flag.ToLower() == "springloadedforwardsimmediately")
+                        {
+                            if (!MouseInputActive)
+                            {
+                                CheckNeutralPosition();
+                                ReloadPositions();
+                            }
+                        }
+                    }
+                }
+            }
+
+        }
+
+        protected void ReloadPositions()
+        {
+            if (noKeyPressed)
+            {
+                foreach (Position pair in PositionsList)
+                {
+                    if (pair.Type == currentPosition)
+                    {
+                        if (pair.Flag.ToLower() == "cruisecontrol.needincreaseafteranybrake")
+                        {
+                            needPowerUpAfterBrake = true;
+                        }
+                        if (pair.Flag.ToLower() == "springloadedforwards" || pair.Flag.ToLower() == "springloadedbackwards")
+                        {
+                            if (elapsedSecondsFromLastChange > 0.2f)
```

Comment:
Constant 0.2f

Reply:
The code seems self-explaining to me here.

Reply:
Yes, the code is clear, but why 0.2 seconds is used and not another value?